### PR TITLE
Using URLFrontier in archetype

### PR DESCRIPTION
**Pull request by jnioche**

### CODE REVIEW

File: archetype/src/main/resources/archetype-resources/README.md [hunk 1]
```diff
@@ -7,7 +7,17 @@ With Storm installed, you must first generate an uberjar:
 mvn clean package
 ```
 
-before submitting the topology using the storm command:
+You need to have an instance of URLFrontier running. See https://github.com/crawler-commons/url-frontier/tree/master/service for instructions; the easiest way is to use Docker.
```
Comment (rzo1) [inline archetype/src/main/resources/archetype-resources/README.md:10]: I suggest to write:

```md
See [the URLFrontier README](https://github.com/crawler-commons/url-frontier/tree/master/service); the easiest way is to use Docker.
```

Comment (rzo1) [reply archetype/src/main/resources/archetype-resources/README.md:10]: Should we link to https://github.com/DigitalPebble/stormcrawler-docker as a resource for an example `docker-compose` to run it straight out of the archetype?

File: archetype/src/main/resources/archetype-resources/README.md [hunk 2]
```diff
@@ -7,7 +7,17 @@ With Storm installed, you must first generate an uberjar:
...
+You will also need to inject URLs into URLFrontier, using the [client](https://github.com/crawler-commons/url-frontier/tree/master/client). Fortunately, it is added as a dependency to this project so all
+you need to do is
+
+``` sh
+java -cp  target/${artifactId}-${version}.jar crawlercommons.urlfrontier.client.Client PutURLs -f seeds.txt
+```
+
+where _seeds.txt_ is a file containing URLs to inject, with one URL per line.
+
+You can now submit the topology using the storm command:
```
Comment (rzo1) [inline archetype/src/main/resources/archetype-resources/README.md:20]: I am wondering, if we should mention, that the user needs a local storm installation for this to work or do we just assume, that a person generating a stormcrawler project via the archetype **knows** that he/she needs a local storm environment setup and link to some reference?

**Review by rzo1 — COMMENTED** [review]
I haven't tested the archetype yet (still no (fast) Internet access on campus) but that is on my todo list. I left some comments - don'T know if these are of much help.

File: archetype/src/main/resources/archetype-resources/README.md (hunk 2 above)
Comment (jnioche) [reply archetype/src/main/resources/archetype-resources/README.md:20]: It says further up in the README 

> With Storm installed, you must first generate an uberjar:

I'll add a  link to the [Setting-up-a-Storm-cluster](https://storm.apache.org/releases/2.4.0/Setting-up-a-Storm-cluster.html)

File: archetype/src/main/resources/archetype-resources/README.md (hunk 1 above)
Comment (jnioche) [reply archetype/src/main/resources/archetype-resources/README.md:10]: The docker-compose setup is useful but more complex to run. I'd rather keep the README for the archetype as simple as possible. I meant to add the commands to start URLFRontier with Docker.
I''ll add the link anyway for reference

**Comment by jnioche** [comment]
> I haven't tested the archetype yet (still no (fast) Internet access on campus) but that is on my todo list. I left some comments - don'T know if these are of much help.

thanks @rzo1, definitely useful. I've improved the README, could you please have a look and let me know what you think of it? Thanks

**Review by rzo1 — COMMENTED** [review]
I like the README enhancement.

**Pull request merged**